src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -106,11 +111,15 @@
    (fn [{:keys [this props state]}]
      (let [nav-context (nav/parse-segment (:nav-context props))
            page (keyword (:segment nav-context))]
-       (when-not (or (contains? (set (map :key top-nav-bar-items)) page)
+       (when-not (or (contains? (set (map :key routes)) page)
                      (= page :status))
          (nav/navigate (:nav-context props) "workspaces"))
        [:div {}
         [:div {:style {:padding "1em" :borderBottom (str "1px solid " (:line-gray style/colors))}}
+         [:div {:style {:float "right" :fontSize "70%"}}
+          [:a {:href "javascript:;"
```

Comment:
This should be `:href "#profile"` rather than having an onClick handler. That way, the user can open in a new tab or whatever they want. This was my bad, but let's grab the opportunity to correct it.